Permanent/ViewControllers/LegacyPlanning/ViewController/LegacyPlanningLoadingViewController.swift

```diff
@@ -16,11 +16,18 @@ class LegacyPlanningLoadingViewController: BaseViewController<LegacyPlanningView
         
         navigationItem.title = "Legacy Planning".localized()
         
-        if viewModel?.getLegacyPlanningAccount() == true {
-            showStatus()
-        } else {
-            showIntro()
-        }
+        viewModel?.getLegacyPlanningAccount(completion: { [weak self] result in
```

Comment:
Should we have a loading spinner here until it gets the steward?

Reply:
In the design of this screen, there is no loading spinner.